modules/framework/galasa-parent/dev.galasa.framework.api.users/src/test/java/dev/galasa/framework/api/users/internal/routes/UsersDeleteRouteTest.java

```diff
@@ -0,0 +1,180 @@
+        authStoreService.addUser(mockUser1);
+
+        // When...
+        servlet.init();
+        servlet.doDelete(mockRequest, servletResponse);
+
+        assertThat(servletResponse.getStatus()).isEqualTo(204);
+    }
+
+    @Test
+    public void testUsersDeletesAUserUsingMeKeywordReturnsOK() throws Exception {
+        // Given...
+        MockEnvironment env = new MockEnvironment();
+        MockTimeService mockTimeService = new MockTimeService(Instant.now());
+        MockAuthStoreService authStoreService = new MockAuthStoreService(mockTimeService);
+        MockFramework framework = new MockFramework(authStoreService);
+
+        String baseUrl = "http://my.server/api";
+
+        String loginId = "me";
+
+        env.setenv(EnvironmentVariables.GALASA_USERNAME_CLAIMS, "preferred_username");
+        env.setenv(EnvironmentVariables.GALASA_EXTERNAL_API_URL,baseUrl);
+        MockUsersServlet servlet = new MockUsersServlet(framework, env);
+
+        MockHttpServletRequest mockRequest = new MockHttpServletRequest("/" + loginId, headerMap);
+        mockRequest.setMethod(HttpMethod.DELETE.toString());
+
+        MockHttpServletResponse servletResponse = new MockHttpServletResponse();     
+        
+        MockUser mockUser1 = createMockUser("testRequestor", "docid", "web-ui");
+        authStoreService.addUser(mockUser1);
+
+        // When...
+        servlet.init();
+        servlet.doDelete(mockRequest, servletResponse);
+
+        assertThat(servletResponse.getStatus()).isEqualTo(204);
+    }
+
+    @Test
+    public void testUsersDeletesAUserAndTheirAccessTokensReturnsOK() throws Exception {
+        // Given...
+        MockEnvironment env = new MockEnvironment();
+        MockTimeService mockTimeService = new MockTimeService(Instant.now());
+        MockAuthStoreService authStoreService = new MockAuthStoreService(mockTimeService);
+        MockFramework framework = new MockFramework(authStoreService);
+
+        String baseUrl = "http://my.server/api";
+
+        String loginId = "user-1";
+
+        env.setenv(EnvironmentVariables.GALASA_USERNAME_CLAIMS, "preferred_username");
+        env.setenv(EnvironmentVariables.GALASA_EXTERNAL_API_URL,baseUrl);
+        MockUsersServlet servlet = new MockUsersServlet(framework, env);
+
+        MockHttpServletRequest mockRequest = new MockHttpServletRequest("/" + loginId, headerMap);
+        mockRequest.setMethod(HttpMethod.DELETE.toString());
+
+        MockHttpServletResponse servletResponse = new MockHttpServletResponse();     
+
+        IInternalUser owner = new InternalUser("user-1", "dexId");
+        authStoreService.storeToken("some-client-id", "test-token", owner);
+        
+        MockUser mockUser1 = createMockUser("user-1", "docid", "web-ui");
+        authStoreService.addUser(mockUser1);
+
+        // When...
+        servlet.init();
+        servlet.doDelete(mockRequest, servletResponse);
+
+        assertThat(servletResponse.getStatus()).isEqualTo(204);
```

Comment:
Same here - have the tokens been removed from the auth store? Have they been revoked correctly?